kimchi/cairo/0001-cairo-gates-and-constraints.md

```diff
@@ -0,0 +1,256 @@
+## Summary
```

Comment:
can you create a separate PR for that RFC (like https://github.com/o1-labs/proof-systems/pull/250/files)